sdk/trace/provider.go

```diff
@@ -91,9 +91,9 @@ func NewProvider(opts ...ProviderOption) (*Provider, error) {
 	return tp, nil
 }
 
-// GetTracer with the given name. If a tracer for the given name does not exist,
+// NewTracer with the given name. If a tracer for the given name does not exist,
```

Comment:
The name 'NewTracer' suggest that it creates a new Tracer. Same issue existed with GetTracer.
How about just Tracer?

Reply:
Sounds better to serve `GetOrCreate` war.

Reply:
I think it's still unclear if the rename makes sense as the spec says "New Tracer instances ... and it's getTracer method", see [here](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#obtaining-a-tracer)

Reply:
@rghetia I personally like `Tracer()`.

I'm reminded of the java.util.logging.Logger API, which supports `getLogger()`. I realize that I endorsed the idea of `NewTracer`, but now I'm thinking about the decoupling of SDK and API, and it's just not a requirement that this method return anything new. So, I'm returning to a preference for `GetTracer()` or just `Tracer()`. Go has approved the idiom of leaving "get" out of simple getters, and semantically, that's all this is. OK, I like `Tracer()`.

Reply:
Further, I believe we have discretion at the language-level to choose appropriate names. We are not bound to use "GetTracer()" because of the specification.

Reply:
But I do believe that New implies a new instance all the time which makes a semantic change

Reply:
As a summary, is the consensus to rename to `Tracer`?

Reply:
yes

Reply:
Updated so.